narwhals/stable/v1.py

```diff
@@ -164,7 +164,8 @@ def to_dict(
         Examples:
             >>> import pandas as pd
             >>> import polars as pl
-            >>> import narwhals.stable.v1 as nw
+            >>> import pyarrow as pa
+            >>> import narwhals as nw
```

Comment:
Hey Vahideh, so cool to see you contribution. 

You can leave: import narwhals.stable.v1 as nw
As we're showing here how you can implement stable version of narwhals